engines/saga/interface.h

```diff
@@ -284,6 +284,8 @@ class Interface {
 
 	int32 getProtectHash() { return _protectHash; }
 
+	PanelModes getPanelMode() const { return (PanelModes)_panelMode; }
```

Comment:
You can use `getMode()` (line 192, above)

Reply:
Fixed